src/main/scala/optics/internal/focus/ErrorHandling.scala

```diff
@@ -0,0 +1,15 @@
+package optics.internal.focus
+
+private[focus] trait ErrorHandling {
+  this: FocusBase => 
+  
+  def errorMessage(error: FocusError): String = error match {
+    case FocusError.NotACaseClass(fromClass) => s"Expecting a case class in the 'From' position; found $fromClass"
```

Comment:
I am very upset about the handling in Scala between the phases being stringly-typed - `scala-clippy` imo was killed by this.